package.json

```diff
@@ -47,7 +43,8 @@
     "ember-try": "^0.2.23",
     "eslint-plugin-ember": "^5.0.0",
     "eslint-plugin-node": "^5.2.1",
-    "loader.js": "^4.2.3"
+    "loader.js": "^4.2.3",
+    "prettier": "1.17.0"
```

Comment:
you might also want to consider adding the following packages:
* `eslint-config-prettier` standard prettier configuration
* `eslint-plugin-prettier` automatically run prettier when eslint runs, so when ember test runs, it' hooks into eslint, and you'll see prettier rules applied 💫

Then in `eslintrc.json`

```js
// change
plugins: ['ember'],
// to 
plugins: ['ember', 'prettier'],
```
and in the rules block, add:
```js
rules: {
    // ... other rules ... 
    // throw errors for prettier format issues
    'prettier/prettier': 2,
    // ... other rules ... 
},
```

Reply:
thanks!

Reply:
@oskarrough you can then run `yarn eslint --fix <path to file to fix>` and eslint/prettier will autoformat it according to your config.

Reply:
I noticed, very neat!